CHANGELOG.rst

```diff
@@ -1,5 +1,35 @@
 .. _changelog:
 
+0.32.0
+------
+
+Renku ``0.32.0`` introduces new features as well as bug fixes for the KG services. 
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**🌟 New Features**
+
+- **KG**: ``updatedAt`` renamed to ``dateModified`` on the Project Details API (`#1582 <https://github.com/SwissDataScienceCenter/renku-graph/issues/1582>`_).
```

Comment:
My question is: is this a new feature or an improvement? The way it is described, looks like an improvement.

Reply:
Indeed, it's an improvement because we are aiming to have all APIs having the same `dateModified` property to give info when things got changed.